client/src/app/tabs/bpmn/__tests__/BpmnEditorSpec.js

```diff
@@ -544,81 +544,63 @@ describe('<BpmnEditor>', function() {
 
   describe('#handleNamespace', function() {
 
-    it('should replace namespace', function(done) {
+    it('should replace namespace', async function() {
 
       // given
       const onContentUpdated = sinon.spy();
       const onAction = sinon.stub().resolves('yes');
 
       // when
-      renderEditor(activitiXML, {
+      await renderEditor(activitiXML, {
         onAction,
         onContentUpdated,
         onImport
       });
 
       // then
       function onImport() {
-        try {
-          expect(onContentUpdated).to.be.calledOnce;
-          expect(onContentUpdated).to.be.calledWith(activitiConvertedXML);
-
-          done();
-        } catch (error) {
-          done(error);
-        }
+        expect(onContentUpdated).to.be.calledOnce;
+        expect(onContentUpdated).to.be.calledWith(activitiConvertedXML);
```

Comment:
In this situation, it is not certain that the expect statements will be run. The callback may run after tests and then any errors are swallowed (false negative).